res/skins/Tango/style.qss

```diff
@@ -2083,11 +2083,56 @@ WTrackProperty#SamplerTitle_mini {
  ######  Misc  ##################################################
 ##############################################################*/
 
-QToolTip {
+QToolTip,
+QMenu {
```

Comment:
Does this still work with #2238? Do you need to style CueMenu and ColorMenu in addition to QMenu?

Reply:
It works, see the screenshot in the PR description which I composed from the hotcue label branch and this branch.
There are a few minor adjustments necessary though.
The LateNight screenshot only shows that it works in general, I'll take care of the other skins and the fine tuning when #2238 is merged